### DISCUSSION TITLE
The problem of irritating the enemy

### DISCUSSION THREAD

**Discussion by 2118557404** [winstxnhdw/lc-hax#212] [Ideas]
There seems to be some bug, the Nutcracker and Little Girl when you are in camera mode, if you see them how far your entity will take damage, and when you use the /hate command when you don't see them<yourself>, you will receive a damage prompt but will not deduct health. When playing against teammates in camera mode, as long as there are two monsters on the map and use the /hate <yourteam>command, the teammates will die directly anywhere. There may also be giants causing teammates to die in the current test camera mode, but if the camera mode is not enabled, this command will also cause teammates to die if there are certain monsters on the map.

**Comment by winstxnhdw** [comment]
I think I fixed this in the latest update.
